tools/view.js

```diff
@@ -440,9 +468,6 @@ View.prototype.displaySuspendEvent = function (data, getSource) {
     var annotationArray = sourceToArray(source.sourceText);
     annotateArray(annotationArray, source.id, data.sections);
     sourceFile.html(arrayToString(annotationArray));
-
-    // enable clicking on EventualSendNodes
-    enableEventualSendClicks(sourceFile);
```

Comment:
Why is this one removed? I think this causes the problem with the breakpoints not working after a breakpoint was triggered